lib/osf-components/addon/components/new-project-modal/template.hbs

```diff
@@ -42,31 +42,33 @@
                         {{t 'new_project.select_all'}}
                     </OsfButton>
                     <table>
-                        <tr data-test-institution-button-row >
-                            {{#each this.institutions as | institution |}}
-                                <td>
-                                    <OsfButton
-                                        data-test-institution-selected='{{if (if-filter institution this.selectedInstitutions) 'selected' 'not-selected'}}'
-                                        data-analytics-name='select_institution'
-                                        local-class='NewProject__institution-button'
-                                        @type='link'
-                                        @onClick={{action this.selectInstitution institution}}
-                                    >
-                                        <div width='45px' height='45px' local-class='NewProject__institution-item'>
-                                            <img
-                                                name='{{institution.name}}'
-                                                width='45px'
-                                                height='45px'
-                                                class='img-circle pointer'
-                                                local-class='NewProject__institution-img-{{if (if-filter institution this.selectedInstitutions) 'selected' 'not-selected'}}'
-                                                src='{{institution.logoRoundedUrl}}'
-                                                alt='{{institution.name}}'
-                                                title='{{institution.name}}'>
-                                        </div>
-                                    </OsfButton>
-                                </td>
-                            {{/each}}
-                        </tr>
+                        <tbody>
+                            <tr data-test-institution-button-row >
+                                {{#each this.institutions as | institution |}}
+                                    <td>
+                                        <OsfButton
+                                            data-test-institution-selected='{{if (if-filter institution this.selectedInstitutions) 'selected' 'not-selected'}}'
+                                            data-analytics-name='select_institution'
+                                            local-class='NewProject__institution-button'
+                                            @type='link'
+                                            @onClick={{action this.selectInstitution institution}}
```

Comment:
Could use element modifier here.

Reply:
I think we want to use `on`  element modifier on HTML elements as opposed to components, because components may have their own way of handling the passed in `click` action.

Reply:
if the only thing the component is doing with the passed-in `onClick` is its own `{{on 'click' @onClick}}` on the element with `...attributes`, i think it'd be better to use `{{on}}` on the component -- it's nice to treat components the same as normal elements, when we can, and we could get rid of the `onClick` arg to simplify the component.

if there's a more complex component with a passed-in action that might be called for different/multiple reasons, then yeah, that should stay an argument -- but i'd probably want a name that indicates that it's more complex than "on click".

Reply:
I suggest we make this part of the ticket enabling the `no-action` rule. 

Element modifiers are not allowed in components with our version of ember-cli-template-lint. We need to upgrade `ember-cli-template-lint` to `^2.0.2`. `ember-cli-template-lint` is now deprecated and gets dropped in Ember 3.17 in favor `ember-template-lint`.